pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -255,6 +256,26 @@ public InternalConfigurationData getInternalConfigurationData() {
         return pulsar().getInternalConfigurationData();
     }
 
+    @GET
+    @Path("/backlogQuotaCheck")
```

Comment:
I don't think we use CamelCase for naming the endpoint. We can rename it to `/backlog-quota-check`.